src/sortByPath.js

```diff
@@ -0,0 +1,54 @@
+import { ascend, identity, map, path, pipe, sortWith, useWith } from 'ramda';
+
+const pathToAscendSort = pipe(path, ascend);
+const mapPathsToAscendSort = map(pathToAscendSort);
+
+/**
+ * Sort a list of objects by a list of paths (if first prop value is equivalent, sort by second, etc).
+ *
+ * @func sortByPath
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.29.0|v2.29.0}
+ * @category List
+ * @sig [[k]] -> [{k: v}] -> [{k: v}]
+ * @param {Array.<string>} paths A list of paths in the list param to sort by
```

Comment:
Maybe `Array.<object>`?

Reply:
The `paths` param is a list of list of strings: `Array.<Array.<string>>`